test-avocado/lib/host_setup.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2013 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+
+# you have to setup domain in virt-manager default network and reboot the machine to apply changes
+# /etc/libvirt/qemu/networks/default.xml
+# add line
+# <domain name='cockpit.lan' localOnly='yes'/>
+unset command_not_found_handle
+HS_BASE_PCKGS="virt-deploy pystache sshpass telnet fabric python-pip avocado virt-manager qemu-img"
+export HS_GRP="virtualization"
+HS_CON="-c qemu:///system"
+export HS_POOLNAME="cockpitx"
+export HS_POOLNAME_PATH=/home/$HS_POOLNAME
+
+function echolog(){
+    echo "INFO: `date '+%Y%m%d-%H:%M:%S'`: $@"
+}
+
+function host_dependencies_fedora(){
+    sudo yum -y yum-plugin-copr
```

Comment:
Yes, but there are two things why I done it that way:
 1. to be able to call host_setup.sh under normal root
 2. in setup.sh there is sudo host_setup, so that sudo is asked only once.

But if you want I'll change it to your suggestion